lib/functions/HttpUtils.ts

```diff
@@ -19,15 +19,15 @@ export async function getJson<T>(URL: string): Promise<T> {
   } else {
     if (response.status && response.status < 400) {
       const json = await response.json();
-      console.log(`Well-knonw response: ${JSON.stringify(json, null, 2)}`);
+      // console.log(`Well-knonw response: ${JSON.stringify(json, null, 2)}`);
       return json as T;
```

Comment:
Casting to T?

Reply:
Yeah, that is a generic, also not part of this PR. @sksadjad Please leave the logs in place. We remove them all at once. We really need those at this point